oshmem/mca/spml/ucx/spml_ucx_component.c

```diff
@@ -128,7 +128,7 @@ static int mca_spml_ucx_component_open(void)
     memset(&params, 0, sizeof(params));
     params.field_mask = UCP_PARAM_FIELD_FEATURES|UCP_PARAM_FIELD_ESTIMATED_NUM_EPS;
     params.features   = UCP_FEATURE_RMA|UCP_FEATURE_AMO32|UCP_FEATURE_AMO64;
-    params.estimated_num_eps = oshmem_num_procs();
+    params.estimated_num_eps = ompi_proc_world_size();
```

Comment:
@alinask - could you please add comment here to describe why mpi call used and not shmem

Reply:
@miked-mellanox the oshmem_num_procs() function uses an unitialized value to get the number of processess in the job so the value returned by this function isn't correct. 
the ompi context was already initialized so using its function instead.
@alex-mikheev what do you think?

Reply:
Why is the value in `osmem_num_procs()` uninitialized -- should that be fixed?

Reply:
oshmem_num_procs() is initialized after spml frameworks are opened.

Reply:
The call to `oshmem_proc_group_init()` which initializes that value is done after the opening and selection of all the components for the job - in `_shmem_init`.